src/api/public/apidocs-new/components/parameters/limit.yaml

```diff
@@ -0,0 +1,6 @@
+in: query
+name: limit
+schema:
+  type: integer
+description: Override limit of entries.
```

Comment:
Hmm, is there are default limit? Otherwise I would not say "override" but maybe more "set limit" or similar

Reply:
I found the default limit here: https://github.com/openSUSE/open-build-service/blob/12bad1ae9bd785f9164f42c4bb8266d12fabd54e/src/backend/bs_srcserver#L5451

That is for the `/search/published/repoinfo/id` endpoint. `/search/published/binary/id` and `/search/published/pattern/id` endpoints have similar code in the backend. I updated the description, that applies for all three endpoints.